src/main/scala/uk/gov/nationalarchives/consignmentexport/S3Files.scala

```diff
@@ -5,32 +5,47 @@ import java.util.UUID
 
 import cats.effect.IO
 import cats.implicits._
+import com.typesafe.config.ConfigFactory
 import org.typelevel.log4cats.SelfAwareStructuredLogger
+import software.amazon.awssdk.services.s3.model.GetObjectResponse
 import uk.gov.nationalarchives.aws.utils.S3Utils
 import uk.gov.nationalarchives.consignmentexport.Main.directoryType
 import uk.gov.nationalarchives.consignmentexport.Utils._
 import uk.gov.nationalarchives.consignmentexport.Validator.ValidatedFileMetadata
-import scala.concurrent.duration._
 
+import scala.concurrent.duration._
 import scala.language.postfixOps
 
 class S3Files(s3Utils: S3Utils)(implicit val logger: SelfAwareStructuredLogger[IO]) {
+  private val configuration = ConfigFactory.load()
```

Comment:
You're loading the config here but you've also added it to the `Configuration` object. The other services like the `KeycloakClient` pass in the `Configuration` object if they need config so it'd be good to do that here and keep things consistent.